backend/src/model.ts

```diff
@@ -70,3 +71,8 @@ export interface ForkTemplate {
   readonly id: number
   readonly url: string
 }
+
+export interface GithubPublicKey {
+  readonly key_id: string
```

Comment:
`keyId`*, niin kun muuallakin. Tai sit kaikki camel_caseen :D

Reply:
Tämä oli sen takia että github itse käyttää key_id. Mutta voidaan laittaa meidän apissa aina camelCasella

Reply:
Aivan 👍🏼 Mut jos tota ei paljasteta meidän APIsta ulospäin niin siinä tapauksessa varmaan ihan sama.